SU-HSDO/suhumsci
# D8CORE-747 Private page content type with a couple paragraphs

# READY FOR REVIEW

# Summary
- Private page content type
- Private files paragraph type
- Text area paragraph type that only allows a wysiwyg without media embedding.

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. `blt drupal:install`
1. log in via `drush uli`
1. create a "Basic private page" content
1. ensure you only have access to 2 paragraph types
1. after creating it, ensure anonymous users can't view the page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
